README.md

```diff
@@ -12,6 +12,18 @@ Scuttlebutt is a decent(ralised) secure gossip platform.
 >
 > Or, if you're lazy: http://en.wikipedia.org/wiki/Scuttlebutt (laziness will get you nowhere, btw)
 
+## Secure-scuttlebutt vs scuttlebutt vs scuttlebot vs sbot?
```

Comment:
i'm open to it being on the home page **if it's a super important**, but definitely closer to the bottom.